compute/admin_guide/agentless-scanning/agentless-scanning-modes.adoc

```diff
@@ -6,20 +6,28 @@ toc::[]
 
```

Comment:
@omri86 Having the content in separate topics makes easier to reference since the landing topic is already very long. Having the multiple topics also improves our SEO.

Reply:
alright, I don't mind it's just not what we discussed :) i'm fine with it